src/lib/network_pool/indexed_pool.mli

```diff
@@ -77,19 +77,19 @@ val empty :
 (** How many transactions are currently in the pool *)
 val size : t -> int
 
-(** What is the lowest fee transaction in the pool *)
-val min_fee : t -> Currency.Fee.t option
+(* The least fee per weight unit of all transactions in the transaction pool *)
+val min_fee : t -> int option
 
-(** Remove the lowest fee command from the pool, along with any others from the
-    same account with higher nonces. *)
+(** Remove the command from the pool with the lowest fee per wu,
+    along with any others from the same account with higher nonces. *)
 val remove_lowest_fee :
   t -> Transaction_hash.User_command_with_valid_signature.t Sequence.t * t
 
 (** Remove all the user commands that are expired. (Valid-until < Current-global-slot) *)
 val remove_expired :
   t -> Transaction_hash.User_command_with_valid_signature.t Sequence.t * t
 
-(** Get the highest fee applicable command in the pool *)
+(** Get the applicable command in the pool with the lowest fee per wu *)
```

Comment:
`highest fee per wu`?

Reply:
'wu' is an acronym for 'weight unit', commonly used when discussing bitcoin-like protocols, eg.

https://www.blockchain.com/btc/block/000000000000000000088421ff58d80d440f207958fe334b6aa6516854e31dd7
https://explorer.b.tc/block/000000000000000000088421ff58d80d440f207958fe334b6aa6516854e31dd7
https://bitcoin.stackexchange.com/questions/89385/is-there-a-difference-between-bytes-and-virtual-bytes-vbytes

I could change 'fee per wu' to the equivalent term 'fee rate'

Reply:
Oh sorry, I meant this should be `highest` instead of `lowest`

Reply:
Fixed in 9a6e6c1310005c479101389cc9fe14d2491ff0cd